pkg/forwarder/transaction.go

```diff
@@ -302,3 +303,11 @@ func (t *HTTPTransaction) internalProcess(ctx context.Context, client *http.Clie
 	log.Tracef("Successfully posted payload to %q: %s", logURL, string(body))
 	return resp.StatusCode, body, nil
 }
+
+// GetPayloadSize returns the size of the payload.
+func (t *HTTPTransaction) GetPayloadSize() int {
+	if t != nil && t.Payload != nil {
+		return len(*t.Payload)
```

Comment:
Why is this a pointer? Why not a plain `[]byte` slice which is pointer-like by nature?

Reply:
This is an old code from https://github.com/DataDog/datadog-agent/commit/4c0da2e763e1c77801a10b4b7e70241e377f3fab#diff-5d59b77d698beef8ea2eee49d67a9302R21.
I do not think there is still a good reason for using `Payload *[]byte`.